packages/fxa-auth-server/test/local/senders/email.js

```diff
@@ -2494,11 +2494,8 @@ describe('email translations', () => {
           'language header is correct'
         );
         // NOTE: translation might change, but we use the subject, we don't change that often.
-        assert.equal(
-          emailConfig.subject,
-          'أكّد حساب فَيَرفُكس الخاص بك',
-          'translation is correct'
-        );
+        // TODO: switch back to testing the subject when translations have caught up
+        assert.include(emailConfig.text, 'سياسة موزيلا للخصوصيّة');
```

Comment:
There were a couple of awkward changes where we're testing translated text, but now we've temporarily broken those assertions until the translators do their thing. For arabic, I couldn't find another matching subject line that was translated, so testing the translation for `Mozilla Privacy Policy` instead.